### ISSUE TITLE
Closing the main window not closes the PowerShell monitor window

### ISSUE THREAD

**Issue by borzaka** [MultiPoolMiner/MultiPoolMiner#1304]
I think it should. In the prev versions if I closed the main window, the miner window was closed as well. Everything was closed fine.

Now the PowerShell monitoring window just waits forever after closing the main window:
`2018-02-23 19:57:39 INFO: Start waiting before next run.`
Not mining, which is OK, I have closed the main window.

And vica-versa: closing the PowerShell monitor window not closes the main window.
Still mining in the background.

V2.7.2.4

**Comment by Gandalf15net** [comment]
I think if you close the main window it stops mining, but powershell window will stay open but inactive. Not sure though.

But a bug for sure

**Comment by mwodzynski** [comment]
I noticed the same issues. 
1. Closing main window (shutting down MPM) does not close monitoring window. It just stays open no mining done. - More of an annoyance than issue
2. Closing monitoring window keeps the miner running without any output data. I believe that monitoring window should restart (open again) since MPM is still working.

In my opinoin it is more of a cosmetic issue - not a priority.
